packages/image_picker/image_picker_ios/example/pubspec.yaml

```diff
@@ -0,0 +1,29 @@
+name: image_picker_example
+description: Demonstrates how to use the image_picker plugin.
+publish_to: none
+
+environment:
+  sdk: ">=2.14.0 <3.0.0"
+  flutter: ">=2.5.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  image_picker_ios:
+    # When depending on this package from a real application you should use:
+    #   image_picker_ios: ^x.y.z
```

Comment:
This stock comment doesn't seem correct, they should depend on `image_picker` no?

Reply:
If someone is explicitly depending on the `image_picker_ios` package, as the example is doing, they should do it like this.

That's not usually what people should do, but nothing in the implementation examples is what people should usually do with a federated plugin. Usually people just use the app-facing package, and would look at the app-facing package's example.

(The implementation package examples are almost entirely for development and test-running purposes. At some point we should maybe batch-update the README boilerplate we use in them to make that more explicit.)